Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: 2200, auto_correct: true
```

Comment:
```
==> centos72: Fixed port collision for 22 => 2200. Now on port 2203.
==> centos72: Fixed port collision for 22 => 2222. Now on port 2204.
==> centos72: Clearing any previously set network interfaces...
==> centos72: Preparing network interfaces based on configuration...
    centos72: Adapter 1: nat
    centos72: Adapter 2: hostonly
==> centos72: Forwarding ports...
    centos72: 22 (guest) => 2203 (host) (adapter 1)
    centos72: 22 (guest) => 2204 (host) (adapter 1)
==> centos72: Running 'pre-boot' VM customizations...
==> centos72: Booting VM...
==> centos72: Waiting for machine to boot. This may take a few minutes...
    centos72: SSH address: 127.0.0.1:2204
```